plugins/woocommerce/client/admin/client/settings-payments/components/payment-settings-layout/payment-settings-layout.scss

```diff
@@ -0,0 +1,22 @@
+.payment-settings-layout {
+	margin: 24px auto 0;
+	max-width: 1032px;
+	display: flex;
+	flex-flow: column;
+
+	// TODO: Add mobile styling (row instead of column).
```

Comment:
spotted that, adding the comment, so it's not missed 🙈

Reply:
Handled in 370a44a843!